src/test/java/com/powsybl/openloadflow/NonImpedantBranchTest.java

```diff
@@ -171,8 +169,8 @@ void inconsistentTargetVoltagesTest() {
         createLine(network, b2, b3, "l23", 0.05);
         createLine(network, b3, b4, "l34", 0.05);
 
-        CompletionException exception = assertThrows(CompletionException.class, () -> loadFlowRunner.run(network, parameters));
-        assertEquals("Non impedant branch 'l12' is connected to PV buses 'b1_vl_0' and 'b2_vl_0' with inconsistent target voltages: 1.0 and 1.01", exception.getCause().getMessage());
+        // CompletionException exception = assertThrows(CompletionException.class, () -> loadFlowRunner.run(network, parameters));
```

Comment:
I have an issue here because I have "removed" this check. It is in the code but never reached. It has to be moved but it could be nice to see that with you @flo-dup. Thanks for help !